store/index.js

```diff
@@ -537,30 +543,22 @@ export const actions = {
     async nuxtClientInit({ commit }) {
         const token = cookies.get('token')
 
-        if (token)
+        if (token) {
             commit('handleToken', { token, save: false })
+            // fetch user if we haven't got it (SPA/static or API error)
```

Comment:
This doesn't need to be a comment. `await this.dispatch('fetchUser')` says what it's doing.